indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -123,6 +122,11 @@ class BookingEditForm extends React.Component {
     return !dt.isSameOrAfter(startDt, 'day');
   };
 
+  getMinDate = () => {
```

Comment:
TODO for me: check if everything important from the `isDateDisabled` logic above is implemented, and remove that function. IIRC i checked it before and it was fine, but I'm not completely sure if i checked it and just forgot to remove it, or missed something and left it there because of that but forgot to add a comment...